src/pudl/etl/__init__.py

```diff
@@ -107,14 +108,17 @@
 }
 
 all_asset_modules = raw_module_groups | core_module_groups | out_module_groups
-default_assets = list(
-    itertools.chain.from_iterable(
-        load_assets_from_modules(
-            modules,
-            group_name=group_name,
+default_assets = (
+    list(
+        itertools.chain.from_iterable(
+            load_assets_from_modules(
+                modules,
+                group_name=group_name,
+            )
+            for group_name, modules in all_asset_modules.items()
         )
-        for group_name, modules in all_asset_modules.items()
     )
+    + get_pudl_models_assets()
```

Comment:
Append assets to load pudl models tables to default set of assets. If `USE_PUDL_MODELS` is not set, then no assets will be added.